packages/core/types/index.d.ts

```diff
@@ -2980,7 +2980,26 @@ export namespace Asciidoctor {
     function create(): Html5Converter;
   }
 
-  class Converter {
+  interface AbstractConverter {
+    /**
+     * Converts an {AbstractNode} using the given transform.
+     * This method must be implemented by a concrete converter class.
+     *
+     * @param node - The concrete instance of AbstractNode to convert.
+     * @param [transform] - An optional String transform that hints at which transformation should be applied to this node.
+     * If a transform is not given, the transform is often derived from the value of the {AbstractNode#getNodeName} property. (optional, default: undefined)
+     * @param [opts]- An optional JSON of options hints about how to convert the node. (optional, default: undefined)
+     *
+     * @returns the {String} result.
+     */
+    convert(node: AbstractNode, transform?: string, opts?: unknown): string;
+  }
+
+  interface ConverterConstructor {
+	new(backend?: string, opts?: unknown): AbstractConverter;
```

Comment:
neat-pick: could you please replace the tab by spaces?

Reply:
Sure!

Reply:
done